package.json

```diff
@@ -1,15 +1,16 @@
 {
   "name": "website",
   "version": "0.0.0",
+  "type": "module",
   "private": true,
   "scripts": {
     "prepare": "husky install",
     "dev": "vitepress dev",
     "build": "vitepress build",
     "preview": "vitepress preview",
-    "create-blog-post": "tsx ./cli/index.mts create-blog-post",
+    "fmt": "prettier",
```

Comment:
Perhaps `format` is better since other script name to be aligned with other scripts?

and why not add `--write` option here?

Reply:
> Perhaps format is better since other script name to be aligned with other scripts?

I mean, other words are not abbreviated. (`dev` is an exception).

Reply:
For the abbreviation of the word, I referred to Deno, Rust (Cargo), and Go, all of which use ‘fmt.’

So, I thought keeping it as ‘fmt’ would be quite standard. What do you think?

Reply:
Regarding the write option, my preference is to use it as a check by default, and I don’t like it automatically writing over things (it’s fine to have writing as the default, but if you add the option, you can’t opt out). What do you think?

Reply:
> For the abbreviation of the word, I referred to Deno, Rust (Cargo), and Go, all of which use ‘fmt.’
> 
> So, I thought keeping it as ‘fmt’ would be quite standard. What do you think?

Oh, I see, then let's make it aligned with ecosystem-level wording rather than project-level 👍 

> Regarding the write option, my preference is to use it as a check by default, and I don’t like it automatically writing over things (it’s fine to have writing as the default, but if you add the option, you can’t opt out). What do you think?

I just wondered if there are cases where we want to run `fmt` script with options, because, for example, we don't have CI step to just check format without fixing. But, yeah, keeping the command "generic" would be reasonable unless our situation is determined. 👍